src/main/java/VOConfigTask.java

```diff
@@ -647,9 +647,16 @@ public String getPilotRoleFQAN() {
          */
         public void setAccountPrefix() throws BuildException {
             if ( (getName() == null) || (getId() == 0) ) {
-                throw new BuildException("VO name or ID undefined: cannot generate account prefix");                
+                throw new BuildException("VO name or ID undefined: cannot generate account prefix");
+            }
+            String voName = getName().replaceFirst("^vo\\.", "").replaceAll("[^A-Za-z0-9]", "");
+            if ( voName.length() >= 3 ) {
+                this.accountPrefix = voName.substring(0,3);
+            } else if ( voName.length() == 2 ) {
+                this.accountPrefix = voName.substring(0,2);
+            } else {
+                throw new BuildException("The VO name is too short: cannot generate account prefix");
             }
-            this.accountPrefix = getName().replaceFirst("^vo\\.", "").replaceAll("[^A-Za-z0-9]", "").substring(0,3);
```

Comment:
I am not sure to understand the intent of this removal. These replace are important to normalize the account prefix. If `substring(0,3)` is the problem when the account prefix is shorter than 3 chars, it is probably enough to replace `3` by the minimum between `3 ` and the length of the VO name after the first replace. Did I miss something?